### ISSUE TITLE
Enhancement: Add H1 tags on all marketing landing pages

### ISSUE THREAD

**Issue by cuemarie** [Automattic/wp-calypso#82485]
### Quick summary

It was reported in slack (p1695974123990299-slack-C029GN3KD) that a couple landpack pages (`/wordpress-free/` and `/wordpress-hosting/`) have titles that are `H2` instead of `H1`. There's concern that this is less accessible for screen readers and other assistive technologies, as well as generally improper HTML semantics.

I checked other Marketing landing pages and found that others do this too, but not all, so there's some discrepancy in the approach.  

Landing pages that use `H2` tags for their title:

- https://wordpress.com/wordpress-free/ (new)
- https://wordpress.com/wordpress-hosting/ 
- https://wordpress.com/hosting/
- https://wordpress.com/domains/
- https://wordpress.com/website-builder/
- https://wordpress.com/create-blog/
- https://wordpress.com/newsletter/
- https://wordpress.com/create-a-course/
- https://wordpress.com/features/

By comparison, the following pages do use an `H1` tag for their title, or hidden somewhere in the structure of the page:

- https://wordpress.com/pricing/
- https://wordpress.com/professional-email/
- https://wordpress.com/website-design-service/
- https://wordpress.com/ecommerce/
- https://wordpress.com/business-name-generator/
- https://wordpress.com/logo-maker/

### Steps to reproduce

1. Open any page on the above list
2. Inspect the title of the page with the browser inspector

### What you expected to happen

All titles to be `H1`, or a consistent practice of using `h1` / `h2`, hidden tags for a11y, etc.

### What actually happened

About half the landpack pages start with `h2` as the top level tag for the title.

### Impact

Some (< 50%)

### Available workarounds?

Yes, easy to implement

**Comment by cuemarie** [comment]
📌 ACTIONS
- Feature request kept
- FYI @Automattic/martech

**Comment by chickee0311** [comment]
Landing pages that use H2 tags for their title have been updated to use H1
